Engine/Tracker/tracker.py

```diff
@@ -47,10 +51,14 @@ def update(self) -> Dict[str, List[Dict[str, Any]]]:
     def _update(self, detection_frame: Dict[str, List[Dict[str, Any]]]):
 
         for robot_obs in detection_frame.get('robots_blue', ()):
+            if TeamColorService().our_team_color == TeamColorService.BLUE and robot_obs['robot_id'] > 7:
```

Comment:
Should probably be a configuration parameter somewhere. On the day we get a eight robot, we will be confused